foundry/run_node.rs

```diff
@@ -241,6 +241,13 @@ pub fn run_node(
         AppDesc::from_str(&app_desc_string).map_err(|err| format!("Foundry failed to parse app descriptor: {}", err))?
     };
     app_desc.merge_params(&module_arguments).map_err(|err| format!("Foundry failed to merge params {}", err))?;
+    let _link_desc = {
+        let link_desc_path = &config.link_desc_path;
+        let link_desc_string = fs::read_to_string(link_desc_path)
+            .map_err(|err| format!("Foundry failed to read link desc file from {}: {}", link_desc_path, err))?;
```

Comment:
`Foundry failed to read a link desc at {}: {}` ?

Reply:
Thanks!